### PR TITLE
Soft copy

### PR DESCRIPTION
**Pull request by cellvia**
add a --soft / -s option for soft copying (do not overwrite existing destination files)

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -69,13 +70,30 @@ function copyFiles(args, config, callback) {
...
       if (pathStat.isFile()) {
-        mkdirp(path.dirname(outName), function (err) {
-          if (err) {
-            return next(err);
-          }
-          next(null, pathName);
-        });
+      	if(soft){
+		  fs.stat(outName, function(err, outStat){
+		  	if(err === null){
```
Comment (calvinmetcalf) [inline index.js]: you should use `!err`, it probably won't be undefined but no reason to to be so specific

**Comment by cellvia** [comment]
cool... done

**Comment by cellvia** [comment]
any other concerns here?  we are pointing directly to my fork but would be nice to point to the actual package

**Comment by calvinmetcalf** [comment]
published in 1.2.0

**Pull request merged**